src/lib/babylon/scene/logic/billboards.ts

```diff
@@ -0,0 +1,10 @@
+import { BillboardMode } from "@dcl/protocol/out-ts/decentraland/sdk/components/billboard.gen";
+
+export function isValidBillboardCombination(billboardMode: BillboardMode) {
+  return (
+    billboardMode == BillboardMode.BM_NONE ||
+    billboardMode == BillboardMode.BM_Y ||
+    billboardMode == (BillboardMode.BM_Y | BillboardMode.BM_X) ||
+    billboardMode == BillboardMode.BM_ALL
```

Comment:
_Question_
Why are Z combinations not valid? Is this something I should add to the annotations in the Billboard `proto` def?

Reply:
It is tricky. We spent many hours experimenting with different rotation schemes and come up with the conclusion that the only usable billboards are Y: trees rotating facing to the camera, YX: a 2D sprite always facing the camera and YXZ: same as YZ but also works on VR when you tilt the head towards a shoulder.

An odd example is `only X`, imagine an arrow that always point to the north (Z-forward). If you apply the `only X` billboard, if you are looking at it from the north, the arrow will copy your camera's inclination. If you are looking from the south, it will copy the camera's inclination but inverted, because it will still be pointing to the north.